upgrade/upgrade-scripts/common.sh

```diff
@@ -142,7 +142,14 @@ function get_image_path() {
 }
 
 function get_image_version() {
-	basename "$(get_image_path)"
+#	basename "$(get_image_path)"
+  source_version_information
+  echo "$VERSION"
+}
+
+function get_image_hotfix() {
```

Comment:
Where is this new function used?

Reply:
Nowhere, was left over from trying mirror existing patterns. Removed.